Update doublecross-south.md (#13)
matthewblain/crosstowntrail-web: cue-sheets/doublecross-south.md

* At Washington, make quick left-right jog onto Waverly Place for one block.
* Right on Clay.
* Left on Hang Ah Alley, passing Willie “Woo Woo” Wong playground.
* Turn right, ascend Sacramento to Powell St.
* Left on Powell.
* Right on California.

> * Right on Cushman St to entrace of Huntington Park near fountain.

Reviewer: ```suggestion
* Right on Cushman St to entrance of Huntington Park mid-block. Enter the park heading towards fountain.
```

> * Cross Huntington Park to Taylor St. and Grace Cathedral steps and turn left to California. (**Note**: Double Cross bonus points for ascending Grace Cathedral steps, and walking labyrinth.)

Reviewer: ```suggestion
* Cross park and exit across from Grace Cathedral steps. Walk left on Taylor to California. (**Note**: Double Cross bonus points for ascending Grace Cathedral steps, and walking labyrinth.)
```
I think it was more clear before.
Alternately,
Cross Huntington Park, exit down steps to Taylor St. Walk left to California across from Grace Cathedral. (**Note**: Double Cross bonus points for ascending Grace Cathedral steps, and walking labyrinth.)
